File: SofiForce.Models/Models/EntityModels/CarTypeModel.cs
using System;

namespace Models
{
    public class CarTypeModel
    {
        public int CarTypeId { get; set; }
        public string CarTypeCode { get; set; }
        public string CarTypeNameEn { get; set; }
        public string CarTypeNameAr { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? IsActive { get; set; }
        public string Color { get; set; }
        public string Icon { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/CarTypeModel.ts
export interface CarTypeModel {
    carTypeId: number;
    carTypeCode: string;
    carTypeNameEn: string;
    carTypeNameAr: string;
    displayOrder: number | null;
    isActive: boolean | null;
    color: string;
    icon: string;
    canEdit: boolean | null;
    canDelete: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}